skaffold.yaml

```diff
@@ -588,6 +588,7 @@ profiles:
   - name: ipv6
     activation:
       - env: IPFAMILY=ipv6
+      - env: IPFAMILY=dual
```

Comment:
This looks confusing at first sight. Shouldn't we rather have a `dual` profile and set `global.admission.service.clusterIPs` value (to be supported in admission chart)?

Reply:
As discussed, the current configuration is fine since we don't need a dual stack service for `gardener-admission-controller`.